07_errors/alextmz/storer_test.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+)
+
+type storerSuite struct {
+	suite.Suite
+	p  []Puppy
+	st Storer
+}
+
+func (su *storerSuite) SetupTest() {
+	su.p = nil
+	var p = []Puppy{
+		{ID: 256, Breed: "Dogo", Colour: "White", Value: 500},
+		{ID: 512, Breed: "Mastiff", Colour: "Brindle", Value: 700},
+		{ID: 768, Breed: "Fila", Colour: "Golden", Value: 900},
+		{ID: 1024, Breed: "Wolfhound", Colour: "Gray", Value: -50},
+	}
+	su.p = append(su.p, p...)
```

Comment:
Nit: Better if you just assign to su.p without appending.

Reply:
Yes I don't know why I did it like that...